lighthouse-cli/cli-flags.js

```diff
@@ -129,7 +129,7 @@ function getFlags(manualArgv) {
         'precomputed-lantern-data-path': 'Path to the file where lantern simulation data should be read from, overwriting the lantern observed estimates for RTT and server latency.',
         'lantern-data-output-path': 'Path to the file where lantern simulation data should be written to, can be used in a future run with the `precomputed-lantern-data-path` flag.',
         'only-audits': 'Only run the specified audits',
-        'only-categories': 'Only run the specified categories',
+        'only-categories': 'Only run the specified categories. Valid categories: accessibility, best-practices, performance, pwa, seo',
```

Comment:
the only tweak I might make is 
```suggestion
        'only-categories': 'Only run the specified categories. Available categories: accessibility, best-practices, performance, pwa, seo',
```

Reply:
if the user has a config that adds other categories those will be valid too, so the slightly looser `Available` makes sense to me too

Reply:
also, sorry the example snippet you edited in the README comes from line 61 of this file if you'd want to update that description too :)

Reply:
yup, good call. I like this better